docs/data/material/customization/css-theme-variables/default-tokens-viewer.md

```diff
@@ -0,0 +1,63 @@
+# Default tokens viewer
+
+<p class="description">A preview of the default CSS theme variables for built-in light and dark color schemes</p>
+
+The following tokens are generated when using `CssVarsProvider` in your application.
+
+## Colors
+
+The color tokens are generated from `colorSchemes`:
+
+{{"component": "modules/material-ui/PaletteTokensViewer.js"}}
+
+## Font
+
+The font tokens are generated from `typography`:
+
+{{"component": "modules/material-ui/FontTokensViewer.js"}}
+
+:::success
+Using font variable is an alternative way to `theme.typography` when you cannot access the theme in Javascript.
```

Comment:
> This applies to every generated CSS variable, not only typography variables, right

Yes, you are right.

> Should we move this info to the top of the file, before the "Colors" section?

I think I will just remove it since this page is about listing the default variables. It might be better to add link to a different page for learning how to use them.